docs/translations/translations.json

```diff
@@ -66,7 +66,8 @@
   "copySourceLinkJS": "Copy link to JavaScript source",
   "copySourceLinkTS": "Copy link to TypeScript source",
   "stackblitz": "Edit in StackBlitz (JS only)",
-  "headTitle": "Material-UI: A React UI component framework",
+  "headTitle": "Material-UI: A popular React UI component framework",
+  "headDescription": "Don't reinvent the wheel. React components for faster and easier web development. Build your own design system, or start with Material Design.",
```

Comment:
First sentence is a bit too strong I feel. The rest is nice though.

Reply:
So just the strapline in this case? "Don't reinvent the wheel." is a generalized criticism of what the industry does. I would fear if people take it personally, yeah.